init.lua

```diff
@@ -63,7 +63,15 @@ local function update(widget, reg, disablecache)
 
     local function update_value(data, t, cache)
         if widget.add_value ~= nil then
-            widget:add_value(tonumber(data) and tonumber(data)/100)
+            if widget.get_stack and widget:get_stack() then
```

Comment:
Just for style consistancy, maybe we should use `widget.get_stack ~= nil`. IMHO for safety we should additionally assert `type(widget.get_stack)` but vicious completely depends on awesome API so I'm not sure if that really matters.